components/resources/jobExplorer/devOpsEngineer/DevOpsProject.tsx

```diff
@@ -1,48 +0,0 @@
-import React from "react";
-const DevOpsProject = () => {
```

Comment:
we don't need this component any more, because we're using the generic re-usable JobExplorerProject component instead. We should slowly delete all of the other project components in favour of using the generic one.